controller/src/index-file.ts

```diff
@@ -45,10 +45,11 @@ export async function addPermissions(
   session: Session,
   index: Index,
   resources: url[],
-  user: UserTypeObject,
+  user: boolean | UserTypeObject,
```

Comment:
I fail to understand how this boolean typing makes sense//why having the user `prop` being optional and passing `null` or `undefined` wouldn't make more sense, but maybe I'm missing something?

Reply:
The idea is as followed:
- `UserTypeObject`: An active user or group
- `boolean`: To indicate access to everyone

I believe that having `null` to indicate public would potentially be less readable?

Reply:
Yeah, but user *assumes* that it's one user. Maybe is it an idea to move this to an new property in the object, e.g. `publicAccess` and define it to true or false and leave the user object null when the `publicAccess` is true?

Reply:
I'm not a huge fan of doing it this way, or is there a way to enforce it that you can't have a `publicAccess: true` and an `UserTypeObject` simultaneously set?

Reply:
After discussing in group we decided to do the user field nullable, with some documentation around it.